tests/bluetooth/tester/src/bttester.h

```diff
@@ -740,6 +740,7 @@ struct l2cap_connect_cmd {
 	uint16_t psm;
 	uint16_t mtu;
 	uint8_t num;
+	bool enhanced;
```

Comment:
Is it really safe to use bool in packed protocol structs? AFAIK it doesn’t have a fixed defined width in the C standard.

Reply:
How is this defined in the BTP specification?

Reply:
I just looked how this is implemented in Mynewt, and I see that what flow control to use is based on number of channels. I think I'll change it accordingly, that way I'll get rid of the bool and this will be easier to handle in autopts

Reply:
@jhedberg Ok, I fixed it, I had to accomodate a few things but it works fine now, and a `bool` is no more